tests/testthat/test-methods.R

```diff
@@ -114,16 +114,15 @@ test_that("print.lint works for inline data, even in RStudio", {
     expect_output(print(l), "not =")
   )
 
+  mockery::stub(print.lints, "rstudioapi::hasFun", function(...) FALSE)
   withr::with_options(
     list("lintr.rstudio_source_markers" = TRUE),
-    with_mock(
-      `rstudioapi::hasFun` = function(...) TRUE,
-      expect_output(print(l), "not =")
-    )
+    expect_output(print(l), "not =")
```

Comment:
test is too vague for me... `not =` doesn't convey all that much. what's the rest of the message here? not clear to me what's being tested exactly.

Reply:
That's part of the lint message printed to console.
A better way may be to mock `print.lint()`, which does the heavy lifting.

Reply:
if the rest of the message is messy, add a comment here like "full message is like '... not= ...'" to give a hint of what should be expected